content/800-data-platform/100-accelerate/580-local-development.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Local development'
+metaTitle: 'Accelerate: Local development'
+metaDescription: 'Learn how to use Prisma Accelerate in a development environment.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma Accelerate is designed to scale database connections efficiently in production environments. Prisma Accelerate also provides a global cache to reduce the load on your database and reduce query response time. To leverage Prisma Accelerate, it is essential to utilize a publicly accessible database.
+
+In development environments, you may want to use a local database to minimize expenses. Furthermore, you may consider extending Prisma Client with the Accelerate client extension once so that you can use a local database in development and a hosted database with Accelerate’s connection pooling and caching enabled. This eliminates the need for conditional logic to switch clients between development and production.
+
+This guide will explain how to use Prisma Accelerate client extension in a development environment with a local database.
+
+</TopBlock>
+
+## Using Prisma Accelerate client extension in development and production
+
+<br />
+<img src="images/accelerate-in-dev.png" alt="" />
+
+Accelerate doesn’t work with a local database. However, using a local database URL, you can add the Accelerate client extension to your Prisma Client. This will allow you to use a local database in your development environment. Keep in mind that, in this scenario, caching and connection pooling won't be available.
+
+Here is how to use a local PostgreSQL database with Prisma ORM and Prisma Accelerate client extension (without caching and connection pooling):
```

Comment:
```suggestion
Here is how to use a local PostgreSQL database with Prisma ORM and Prisma Accelerate client extension:
```

Since that was mentioned in the previous paragraph, we can remove it